### DISCUSSION TITLE
Error: Downloaded file is either not a valid upgrade package or corrupted ?

### DISCUSSION THREAD

**Discussion by carlbeechvuw** [community/community#38737] [Enterprise]
### Select Topic Area

Question

### Body

Hi

We have github enterprise running on-site, and on the latest update I'm getting the error 'Error: Downloaded file is either not a valid upgrade package or corrupted' when running ghe-update-check...

To me it appears the 3.6.3 update package is corrupted?   

Can anyone verify please?

Many thanks

Carl.

**Comment by chris-gillatt** [comment]
I had this exact same issue today, and I think it was due to the automatically downloaded update becoming corrupted for whatever reason.  To resolve, I removed the tmp file and performed a manual download and it verified successfully;

1. `sudo rm /var/lib/ghe-updates/github-enterprise-gce-3.6.3.pkg.tmp`
2. `ghe-update-check -i`

```
Package verified successfully.
```

**Reply by carlbeechvuw** [reply]
Hi 

Many thanks - thats managed to download and install 3.7.0..

Do you know if 3.7 is significantly larger than 3.6?  - while its downloaded, its failed because of insufficient disk space (needs 145Gb )...

Thanks

Carl.

**Comment by chris-gillatt** [comment]
Yeah the minimum recommended volume is now 200GB.
